ros/src/aero_telemetry_simple/src/aero_drone_telemetry.cpp

```diff
@@ -35,11 +35,12 @@ void AeroDrone::fetchTelemetryThread()
   // Battery state
   auto battery_state_sub = nh_.subscribe<sensor_msgs::BatteryState>(
       "mavros/battery", 1, boost::bind(&AeroDrone::setBatteryStateCB, this, _1));
+  ros::Rate rate(10);
```

Comment:
I think you can remove `rate_` from [AeroDrone class](https://github.com/intel-aero/sample-apps/blob/master/ros/src/aero_telemetry_simple/include/aero_drone_telemetry.h#L45).